web/components/shared/themed/themedTabs.tsx

```diff
@@ -46,7 +46,9 @@ export default function ThemedTabs(props: ThemedTabsProps) {
                 <div className="flex flex-row space-x-2 items-center w-fit">
                   <option.icon
                     className={clsx(
-                      selected ? "text-sky-500" : "text-gray-500",
+                      selected
+                        ? "text-sky-500 dark:text-orange-500"
```

Comment:
lets keep the coloring the same for now. between light/dark mode we should keep everything consistent. 500 works for both

Reply:
Do you mean 'orange' or 'blue' coloring? I changed it to orange because graph is 'orange'
![Screenshot 2023-12-11 at 11 05 27 AM](https://github.com/Helicone/helicone/assets/39514133/82d1eacf-a5e4-4ad3-970b-d8214b62c76d)
![Screenshot 2023-12-11 at 11 05 35 AM](https://github.com/Helicone/helicone/assets/39514133/597e4137-14d0-41b9-9e73-ccc74f3a642a)

Reply:
Switched to blue for consistency